_infra/helm/locust/locustfiles/Pipfile

```diff
@@ -0,0 +1,10 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+[packages]
+google-cloud-storage = "*"
```

Comment:
The locust helm chart installs this google package but we'll install it explicitly into the pipenv.
```
    pip_packages:
      - google-cloud-storage
```